README.md

```diff
@@ -1,12 +1,14 @@
 ## Hello there :suspect:, I am Paul Artushkov (also known as pavTiger)
 Since early childhood, I've been interested in technology. I started with soldering some boards and Arduino components. 
-Then I slowly switched to Robotics olympiads with LEGO EV3 system. And went to RRO (Russian Robot Olympiad) in <b>Innopolis</b>. 
-As I went to the fifth grade, I switched to more practical programming (that was the time when I started using <b>GitHub</b>). 
-Enjoy hackathons very much and have been to many of <b>CROC<b> hacks (and won a few). Recently, I started coding Olympiad tasks in <b>Tinkoff Generation<b>.
+Then I slowly switched to Robotics olympiads with LEGO EV3 system. And went to RRO (Russian Robot Olympiad) in **Innopolis**. 
+As I went to the fifth grade, I switched to more practical programming (that was the time when I started using **GitHub**). 
+Enjoy hackathons very much and have been to many of **CROC** hacks (and won a few). Recently, I started coding Olympiad tasks in **Tinkoff Generation**.
 
 I study in an IT & Physics school named Silaeder, where we do many projects and hackathons.
 
-Good-known languages: Python, C++, Javascript, Pascal
+Well-known languages: Python, C++, Javascript, Pascal, C.  
```

Comment:
Again, note that "Well-known" does not mean "Those that I know well". It means "Those that most people heard about". It is sort of a synonym of famous.